core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -350,6 +356,13 @@ object SparkEnv extends Logging {
         "levels using the RDD.persist() method instead.")
     }
 
+    val outputCommitCoordinator = mockOutputCommitCoordinator.getOrElse {
+      new OutputCommitCoordinator(conf)
+    }
```

Comment:
should this only be initialized if `isDriver` is true? It seems that we don't need this on the executors